java-manta-client-unshaded/src/main/java/com/joyent/manta/client/RecursiveDirectoryCreationStrategy.java

```diff
@@ -20,8 +20,8 @@
 /**
  * Utility class for recursive directory creation strategies.
  *
- * @author <a href="https://github.com/tjcelaya">Tomas Celayac</a>
- * @since 3.1.7
+ * @author <a href="hhttps://github.com/douglasAtJoyent">Douglas Anderson</a>
+ * @since 3.2.4
```

Comment:
I think you mixed up RecursiveDirectoryCreationStrategy.java and PruneEmpytParentDirectoryStrategy.java when making this change

Reply:
Sorry, will revert and change the correct file.